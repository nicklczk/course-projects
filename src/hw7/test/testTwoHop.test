LoadGraph champions2 champions2.tsv

FindPath champions2 A D