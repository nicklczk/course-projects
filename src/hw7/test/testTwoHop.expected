loaded graph champions2

path from A to D:
A to C with weight 0.250
C to D with weight 0.333
total cost: 0.583